test-suite/testkit/server/package.json

```diff
@@ -8,7 +8,7 @@
   },
   "private": true,
   "scripts": {
-    "proto": "pbjs -t static-module -r test ../../../examples/cryptocurrency/src/proto/service.proto ../../../components/crypto/src/proto/schema/exonum/crypto/types.proto -o ./test/stubs.js",
+    "proto": "pbjs -t static-module -r test -p node_modules/exonum-client/proto ../../../examples/cryptocurrency/src/proto/service.proto -o ./test/stubs.js",
```

Comment:
Is it intentional or accidental change?

Reply:
It's intentional. The description is [here](https://github.com/exonum/exonum/pull/1770#discussion_r376362198).